twitter/statuses.go

```diff
@@ -12,6 +12,7 @@ import (
 // Deprecated fields: Contributors, Geo, Annotations
 type Tweet struct {
 	Coordinates          *Coordinates           `json:"coordinates"`
+	FullText             string                 `json:"full_text"`
```

Comment:
Isn't full_text intended to be a subfield of extended_tweet? What tweet did you test this on?